search/django/weedid/utils.py

```diff
@@ -179,17 +190,3 @@ def parse_category_name(category):
             "role": "",
             "scientific_name": "",
         }
-
-
-def validate_email_format(email):
```

Comment:
This removal might be unintentional

Reply:
these 2 methods are duplicate for some reason. I'll remove it from another pr